pkg/logger/redis.go

```diff
@@ -0,0 +1,28 @@
+package logger
+
+import (
+	"context"
+
+	"github.com/scribd/go-sdk/pkg/instrumentation"
+)
+
+type (
+	RedisLogger struct {
+		logger Logger
+	}
+)
+
+func NewRedisLogger(l Logger) *RedisLogger {
+	return &RedisLogger{l}
+}
+
+func (r *RedisLogger) Printf(ctx context.Context, format string, v ...interface{}) {
```

Comment:
@fotos Well, we can't change that since it is the logger interface of the underlying [go-redis](https://github.com/redis/go-redis/blob/9f1522a91e666ee40f807945e49396b09e4f87c2/internal/log.go#L10-L12) library 😅

Reply:
Got it.